lib/js/atoms/Stepper.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div class="stepper">
+		<div v-for="(step, index) in steps" :key="index" class="stepper__item">
+			<div
+				v-if="index !== 0"
+				class="stepper__separator"
+				:class="{ '-active': step.isActive }"
+			></div>
+			<div class="stepper__icon" :class="{ '-active': step.isActive }">
+				<icon :icon="ICONS[step.iconKey]" :size="ICON_SIZES.X_SMALL"></icon>
+			</div>
+			<div class="stepper__label" :class="{ '-active': step.isActive }">
+				{{ step.label }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../styles/settings/spacings';
+@import '../../styles/settings/colors';
+@import '../../styles/settings/media-queries';
+@import '../../styles/settings/typography';
+
+.stepper {
+	display: flex;
+
+	&__item {
+		align-items: center;
+		display: flex;
+		justify-content: center;
+	}
+
+	&__label {
+		@include textInfoM();
+		@include textBold();
+
+		color: $color-minor;
+
+		&.-active {
+			color: $color-primary;
+		}
+	}
+
+	&__icon {
+		align-self: center;
+		background-color: $color-minor-background;
+		border-radius: 100%;
+		color: $color-minor;
+		display: flex;
+		height: 32px;
+		justify-content: center;
+		margin-right: $space-xxs;
+		width: 32px;
+		transition: background-color 0.33s;
+
+		&.-active {
+			background-color: $color-primary-background;
+			color: $color-primary;
+		}
+	}
+
+	&__separator {
+		background-color: #e6e8ee;
+		border-radius: 2.5px;
+		height: 2px;
+		margin: 0 $space-xs;
+		transition: background-color 0.33s;
+		width: 32px;
```

Comment:
it shouldn't be fixed width. We can define mix and max, but definitely it should be adjusting based on the number of steps and window size

Reply:
Ewa is gonna to send us exact requirements, however we don't need it right now